controlplane/src/core/bufservices/user/removeOrganizationMember.ts

```diff
@@ -91,12 +91,13 @@ export function removeOrganizationMember(
     }
 
     await opts.keycloakClient.authenticateClient();
-
-    await opts.keycloakClient.removeUserFromOrganization({
-      realm: opts.keycloakRealm,
-      userID: user.id,
-      groupName: org.slug,
-    });
+    if (org.kcGroupId) {
```

Comment:
we should return an error if the kcGroupId doesnt exist right? Because, if it doesnt exist it will tell the admin that the user is removed but he/she is not removed from keycloak

Reply:
Yep, what I meant was to to return if it doesnt